resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+    - [Create Cloud Router interfaces and BGP peers](#create-cloud-router-interfaces-and-bgp-peers)
+- [Configure the \<vendor-name>\<vendor product> side](#configure-the-vendor-namevendor-product-side)
+    - [Creating the base network configuration](#creating-the-base-network-configuration)
+    - [Creating the base VPN gateway configuration](#creating-the-base-vpn-gateway-configuration)
+    - [GCP-compatible settings for IPSec and IKE](#gcp-compatible-settings-for-ipsec-and-ike)
+        - [Configure the IKE proposal and policy](#configure-the-ike-proposal-and-policy)
+        - [Configure the IKEv2 keyring](#configure-the-ikev2-keyring)
+        - [Configure the IKEv2 profile](#configure-the-ikev2-profile)
+        - [Configure the IPsec security association (SA)](#configure-the-ipsec-security-association-sa)
+        - [Configure the IPsec transform set](#configure-the-ipsec-transform-set)
+        - [Configure the IPsec static virtual tunnel interface (SVTI)](#configure-the-ipsec-static-virtual-tunnel-interface-svti)
+    - [Configuring the dynamic routing protocol](#configuring-the-dynamic-routing-protocol)
+    - [Saving the configuration](#saving-the-configuration)
+    - [Testing the configuration](#testing-the-configuration)
+- [Troubleshooting IPsec on \<vendor-name>\<product-name>](#troubleshooting-ipsec-on-vendor-nameproduct-name)
+- [Reference documentation](#reference-documentation)
+    - [GCP documentation](#gcp-documentation)
+    - [\<vendor-name>\<product-name> documentation](#vendor-nameproduct-name-documentation)
+
+<Put trademark statements here>: <vendor terminology> and the <vendor> logo are
+trademarks of <vendor company name> or its affiliates in the United States
+and/or other countries.
+
+_Disclaimer: This interoperability guide is intended to be informational in
+nature and shows examples only. Customers should verify this information by
+testing it._
+
+Author: <author name and email address>
+
+## Introduction
+
+This guide walks you through the process of configuring
+\<vendor-name>\<product-name> for integration with the
+[HA VPN service](https://cloud.google.com/vpn/docs) on GCP.
+
+For more information about HA or Classic VPN, see the
+[Cloud VPN Overview](https://cloud.google.com/compute/docs/vpn/overview).
+
+## Terminology
+
+Below are definitions of terms used throughout this guide.
+
+<This is some sample terminology. Add any terminology to this section that needs
+explanation.>
+
+-  **GCP VPC network**—A single virtual network within a single GCP project.
+-  **On-premises gateway**—The VPN device on the non-GCP side of the
+connection, which is usually a device in a physical data center or in
+another cloud provider's network. GCP instructions are written from the
+point of view of the GCP VPC network, so "on-premises gateway" refers to the
+gateway that's connecting _to_ GCP.
+-  **External IP address** or **GCP peer address**—a single static IP address
```

Comment:
With HA VPN , one external address for each HA VPN interface are automatically created when you create the gateway. You don't create External IPs manually as you do for Classic VPN.

Reply:
I am thinking to remove this section. We are explaining all the terms in the table below (Configuration parameters and values section) already. Thoughts ?

Reply:
The explanations aren't entirely the same. If you can fit them into the table, that might work.